hack/setup-demos-ci.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+#
+# Refer to the --help output for a description of this script and its available options.
+#
+
+infomsg() {
+  echo "[INFO] ${1}"
+}
+
+helpmsg() {
+  cat <<HELP
+This script will run setup a set of demos for testing Kiali against a real environment in CI.
+HELP
+}
+
+# abort on any error
+set -e
+
+# set up some of our defaults
+
+infomsg "Make sure everything exists"
+which kubectl > /dev/null || (infomsg "kubectl executable is missing"; exit 1)
```

Comment:
To be able to reuse this same hack script for testing on openshift, this hack script should work with `oc` as well.

Reply:
There is a specific hack for downstream:
```
hack/istio/install-error-rates-demo.sh
```

As the pipelines won't be the same, the downstream pipeline can just use that script I think.

(In the same way that the kind script is only used for upstream pipelines).

But tests should work in both environments.

Reply:
I'm reviewing that hack script and it's prepared for both environments.
I'm going to test it and update the PR.

Reply:
If the `install-error-rates-demo` script already does all the necessary setup for both upstream/downstream, why not just call that hack script from this hack script? It looks like `install-error-rates-demo` will work for both.

> As the pipelines won't be the same, the downstream pipeline can just use that script I think.

What the environment is running on will be different, I don't think downstream runs on kind, but the environment will be roughly the same. Openshift might require a couple extra objects to be created like the network definitions but it looks like the `install-error-rates-demo` already handles that. If we can use the same setup scripts for both environments, we're less likely to diverge between the two.

Reply:
PR updated.
Using the `install-error-rates-demo.sh` script and it works well, no need to duplicate this hack.
I've removed the new `setup-demos-ci.sh` one.